app/src/main/res/values-land/dimens.xml

```diff
@@ -14,6 +14,7 @@
     <dimen name="video_item_search_thumbnail_image_width">142dp</dimen>
     <dimen name="video_item_search_thumbnail_image_height">80dp</dimen>
     <!-- Calculated: 2*video_item_search_padding + video_item_search_thumbnail_image_height -->
+    <dimen name="video_item_search_padding">18dp</dimen>
```

Comment:
What is this? Also, pay attention to the comment in the line above (16): I think it refers to the line below (18), not to this one (15)